compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFinder.java

```diff
@@ -1595,6 +1609,19 @@ private boolean isWithinNodeMetaData(TopLevelNode node) {
             }
         }
 
+        if (node instanceof DocumentableNode) {
```

Comment:
Need to rethink this. **This is only working by chance, not design**.

Reply:
Updated the logic which prevented from reaching this code segment.